src/preferences/dialog/dlgprefcolors.cpp

```diff
@@ -47,12 +54,44 @@ void DlgPrefColors::loadSettings() {
         comboBoxTrackColors->addItem(paletteName);
     }
 
+    const ColorPalette hotcuePalette = m_colorPaletteSettings.getHotcueColorPalette();
+
     comboBoxHotcueColors->setCurrentText(
-            m_colorPaletteSettings.getHotcueColorPalette().getName());
+            hotcuePalette.getName());
     comboBoxTrackColors->setCurrentText(
             m_colorPaletteSettings.getTrackColorPalette().getName());
 
-    slotApply();
+    QPixmap pixmap(80, 80);
+    QPainter painter(&pixmap);
+    pixmap.fill(Qt::black);
+
+    comboBoxHotcueDefaultColor->addItem(tr("By hotcue number"), -1);
+
+    for (int i = 0; i < hotcuePalette.size() && i < 4; ++i) {
+        painter.setBrush(mixxx::RgbColor::toQColor(hotcuePalette.at(i)));
+        painter.drawRect(0, i * 20, 80, 20);
+    }
+    comboBoxHotcueDefaultColor->setItemIcon(0, QIcon(pixmap));
+
+    for (int i = 0; i < hotcuePalette.size(); ++i) {
+        comboBoxHotcueDefaultColor->addItem(tr("Palette") +
```

Comment:
"Palette N" is not the right term, why no use something like "Color N: #FF8000"?